auth/authenticate.js

```diff
@@ -0,0 +1,43 @@
+const jwt = require('jsonwebtoken');
+
+const jwtKey =
+  process.env.JWT_SECRET ||
+  'add a .env file to root of project with the JWT_SECRET variable';
+
+// quickly see what this file exports
+module.exports = {
+  authenticate,
+  generateToken
+};
+
+function generateToken(user) {
```

Comment:
If you are only going to use one thing from the the object that you are passing in then you can just destructure it as a parameter
```suggestion
function generateToken({ username }) {
```